Networking/Networking/Model/TaxClass.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// Represent a Tax Class Entity.
+///
+public struct TaxClass: Decodable {
```

Comment:
What do you think about adding the `siteID` as an attribute? Since TaxClass is a separate entity, it might be useful to have the siteID as a primary key. For example, it would be similar to `OrderStatus`.

Without the siteID, all TaxClass items would be stored without a way to group and retrieve them separately.

Reply:
Good question.
The reason why I didn't use `siteID` is that I assumed that every time a user switches between store the DB is erased. But probably not so, am I wrong?

Reply:
Switching stores does not erase the entire DB. There's a long conversation in p99K0U-1Fs-p2, but the tl;dr: is that we should align with what Android is currently doing. Products have a siteID and only delete on log out, not when the user switches between sites.

Reply:
I learned something new, thanks @mindgraffiti! I will also add `siteID`. 😉